tests/test_builds.py

```diff
@@ -526,7 +526,7 @@ def test_empty(tmp_path):
     file = testproject_path / "site/empty/index.html"
     contents = file.read_text(encoding="utf8")
 
-    validate_additional_script_code(contents, exists=True)
+    validate_additional_script_code(contents, exists=False)
```

Comment:
On `plugin.py:L132` I'm deliberately skipping any processing if it doesnt contain a tag.
Is it necessary for pages that do not contain swagger tags to have these additional scripts?